front/app/containers/EventsShowPage/components/MetadataInformation/EventTimeTextual.tsx

```diff
@@ -21,7 +21,7 @@ const FullEventTime = ({ event }: Props) => {
     <Container>
       <StyledIcon name="calendar" ariaHidden />
       <Content>
-        <Text my="4px" color="coolGrey600" fontSize="s">
+        <Text my="4px" color="coolGrey600" fontSize="s" aria-hidden>
```

Comment:
Could you add a comment on why it's `aria-hidden`? It's not super close to the others so it may not be clear.